packages/SystemUI/src/com/android/systemui/screenshot/ScreenshotController.java

```diff
@@ -854,8 +854,11 @@ private void playCameraSound() {
      * failure).
      */
     private void saveScreenshotAndToast(Consumer<Uri> finisher) {
-        // Play the shutter sound to notify that we've taken a screenshot
-        playCameraSound();
+        if (Settings.System.getInt(mContext.getContentResolver(),
```

Comment:
Better to move the check inside `playCameraSound()`

Reply:
Moved the check